src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -19,11 +22,15 @@ public class PythonLanguageService : LanguageProcessor
         private readonly string _pythonExecutablePath;
         public override string ProcessName => _pythonExecutablePath;
         private readonly string _apiScriptPath;
+        private readonly string _tempDiirectory;
+
+        static TelemetryClient _telemetryClient = new TelemetryClient(TelemetryConfiguration.CreateDefault());
 
         public PythonLanguageService(IConfiguration configuration)
         {
             _pythonExecutablePath = configuration["PYTHONEXECUTABLEPATH"] ?? "python";
             _apiScriptPath = Path.Combine(Path.GetDirectoryName(typeof(PythonLanguageService).Assembly.Location), "api-stub-generator", "apistubgen.py");
+            _tempDiirectory = Path.Combine(Path.GetDirectoryName(typeof(PythonLanguageService).Assembly.Location), "temp");
```

Comment:
Why use this instead of the user temp path? Is there some permission issue?

Reply:
Removed this change and reverted to use user temp path